tests/base/test_regularization.py

```diff
@@ -5,6 +5,8 @@
 import inspect
 
 TOL = 1e-20
+testReg = True
```

Comment:
the changes to the code that follow are tabs because we added these to allow switches for testing